Vagrantfile

```diff
@@ -15,33 +15,40 @@ BEGIN {
 }
 
 # netplugin_synced_gopath="/opt/golang"
-go_version = ENV["GO_VERSION"] || "1.7.5"
-docker_version = ENV["CONTIV_DOCKER_VERSION"] || "1.12.6"
-docker_swarm = ENV["CONTIV_DOCKER_SWARM"] || "classic_mode"
-gopath_folder="/opt/gopath"
+go_version = ENV['GO_VERSION'] || '1.7.5'
+docker_version = ENV['CONTIV_DOCKER_VERSION'] || '1.12.6'
+docker_swarm = ENV['CONTIV_DOCKER_SWARM'] || 'classic_mode'
+gopath_folder = '/opt/gopath'
 http_proxy = ENV['HTTP_PROXY'] || ENV['http_proxy'] || ''
 https_proxy = ENV['HTTPS_PROXY'] || ENV['https_proxy'] || ''
 build_version = ENV['BUILD_VERSION'] || ''
-
-cluster_ip_nodes = ""
+cluster_ip_nodes = ''
+v2plugin_name = ENV['CONTIV_V2PLUGIN_NAME'] || 'contiv/v2netplugin:0.1'
```

Comment:
@rhim: This piece was exactly like this before. The v2plugin isn't released and there's no available version right now.